THIRD_PARTY.md

```diff
@@ -1,27 +1,27 @@
 
-### Kotlin
+## In the OUDS Android libraries
 
-Copyright 2010-2024 JetBrains s.r.o. and Kotlin Programming Language contributors
-[https://github.com/JetBrains/kotlin](https://github.com/JetBrains/kotlin)
-Apache License 2.0
+| Library                                                                                   | Copyright                                                                   | License            |
+|-------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|--------------------|
+| [AndroidX](https://github.com/androidx/androidx)                                          | (c) 2021 The Android Open Source Project                                    | Apache License 2.0 |
+| [Dokka](https://github.com/Kotlin/dokka)                                                  | (c) 2014-2019 JetBrains s.r.o. and Dokka project contributors               | Apache License 2.0 |
+| [Kotlin](https://github.com/JetBrains/kotlin)                                             | (c) 2010-2024 JetBrains s.r.o. and Kotlin Programming Language contributors | Apache License 2.0 |
+| [Material Components](https://github.com/material-components/material-components-android) | (c) 2017 The Android Open Source Project                                    | Apache License 2.0 |
+| [Mockito-Kotlin](https://github.com/mockito/mockito-kotlin)                               | (c) 2016 Niek Haarman<br/>(c) 2007 Mockito contributors                     | MIT License        |
+| [Paparazzi](https://github.com/cashapp/paparazzi)                                         | (c) 2019 Square, Inc.                                                       | Apache License 2.0 |
 
-### Material Components
-
-Copyright 2017 The Android Open Source Project
-[https://github.com/material-components/material-components-android](https://github.com/material-components/material-components-android)
-Apache License 2.0
 
 ## In the Design System Toolbox app
 
-### OUDS libraries
-
-Copyright (c) Orange SA
-[https://github.com/Orange-OpenSource/ouds-android](https://github.com/Orange-OpenSource/ouds-android)
-MIT License
+| Library                                                               | Copyright               | License            |
+|-----------------------------------------------------------------------|-------------------------|--------------------|
+| [Firebase](https://github.com/firebase/firebase-android-sdk)          | (c) 2019 Google LLC     | Apache License 2.0 |
+| [Git Changelog Lib](https://github.com/tomasbjerre/git-changelog-lib) |                         | Apache License 2.0 |
+| [OUDS Android](https://github.com/Orange-OpenSource/ouds-android)     | (c) 2024-2025 Orange SA | MIT License        |
+| [ZXing](https://github.com/zxing/zxing)                               |                         | Apache License 2.0 |
```

Comment:
I found several files with a copyright, but the year is not always the same. For instance `Copyright 2009 ZXing authors`. Should we fill the copyright column with `© ZXing authors`?